components/math/MultiplicationBlock.tsx

```diff
@@ -4,22 +4,26 @@ export interface MultiplicationBlockProps {
   text: string;
   selected: boolean;
   onClick: boolean;
+  player: boolean;
 }
 
 export const MultiplicationBlock: React.FC<MultiplicationBlockProps> = ({
   text,
   onClick,
+  player,
   ...props
 }) => {
   const [selected, setSelected] = useState(false);
+
   const handleSelected = () => {
     setSelected(!selected);
   };
+
   return (
     <div>
       <ul
         className={`w-20 h-20 flex justify-center items-center border-2 text-green-50 float-left
-        ${!selected ? "bg-green-400" : "bg-red-400"}`}
+        ${!selected ? "bg-green-400" : "bg-red-400"} `}
```

Comment:
What if you had a nested ternary here? Could you determine the right colour based on the values of both selected and player?